cppguide.html

```diff
@@ -1594,50 +1599,30 @@ <h3 id="Access_Control">Access Control</h3>
 be <code>protected</code> when using
 
 
-<a href="https://code.google.com/p/googletest/">Google
+<a href="http://code.google.com/p/googletest/">Google
```

Comment:
Do we really want to drop https? Double check other URLs in this edit. Also this particular URL should be https://github.com/google/googletest.